stubs/EntityManagerInterface.stub

```diff
@@ -60,6 +60,14 @@ interface EntityManagerInterface extends ObjectManager
 	 */
 	public function copy($entity, $deep = false);
 
+    /**
+     * @return void
+     *
+     * @throws ORMException
+     * @throws \Doctrine\DBAL\Exception\UniqueConstraintViolationException
+     */
+    public function flush();
```

Comment:
Please fix indenting of `flush` and `getClassMetadata` in this file, thanks.

Reply:
It should be better now